src/actions/Dashboard/index.js

```diff
@@ -28,13 +28,13 @@ function fetchCommonTerms(settings, callback, timespanType, fromDate, toDate, ca
     const dataSources = toDataSources((streams && streams.streams) || []);
 
     DashboardServices.getCommonTerms(timespanType, fromDate, toDate, configuration.targetBbox, configuration.defaultZoomLevel, category, 
-        (error, response, body) => ResponseHandler(error, response, body, (err, topics) => {
-            if (!err) {
-                callback(null, Object.assign({}, { terms, dataSources }, { configuration }, topics ));
-            } else {
-                callback(err, null);
-            }
-        }));
+      (error, response, body) => ResponseHandler(error, response, body, (err, topics) => {
+          if (!err) {
+              callback(null, Object.assign({}, { terms, dataSources }, { configuration }, topics ));
```

Comment:
Nit: can replace `Object.assign` with object literal `{ terms, dataSources }`.

Reply:
I didn't write this file, so to be safe I am going to leave it as is, just to make sure I don't introduce a breaking change.